project/Build.scala

```diff
@@ -1216,8 +1226,50 @@ object Build extends sbt.Build {
     base = file("test-suite/js"),
     settings = commonSettings ++ myScalaJSSettings ++ testTagSettings ++
       testSuiteCommonSettings(isJSTest = true) ++ Seq(
-        name := "Scala.js test suite",
+        name := "Scala.js test suite"
+      ) ++ inConfig(Test)(Seq(
+        artifactPath in jsTestDefinitions :=
+          crossTarget.value / "test_definitions.js",
+
+        // Generate a JS file that contains defined tests and testFrameworks.
+        // The file will be called "test_definitions.js" and defines:
+        // - var definedTests
+        // - var testFrameworkNames
+        jsTestDefinitions := {
```

Comment:
Yes, indeed. But I want us to get some experience first in using this HTML runner. Eventually, this should indeed go into the sbt plugin.